libraries/eslint-config/src/index.js

```diff
@@ -85,6 +85,9 @@ const rules = {
   // Custom rules
   '@bluedot/custom/no-default-tailwind-tokens': ['error'],
   '@bluedot/custom/no-overflow-scroll': ['error'],
+
+  // React hooks rules
+  'react-hooks/set-state-in-effect': ['off'],
```

Comment:
[This](https://react.dev/reference/eslint-plugin-react-hooks/lints/set-state-in-effect) is more of a pain to adopt, because this is usually quite integral to the logic where it is used. I'm also sceptical of this one being bad most of the time